adage/adage/config.py.template

```diff
@@ -39,25 +43,26 @@ OS_CONFIG = {
     'home_dir':      '/home/<configure:default=adage>/',
 }
 
+
 # data for bootstrapping the database and testing
 DATA_CONFIG = {
-    'data_dir':      OS_CONFIG['home_dir'] + 'adage-server/data/',
+    'data_dir':      os.path.join(REPO_ROOT, 'data'),
 }
 DATA_CONFIG.update({
-    'annotation_file':  DATA_CONFIG['data_dir'] +
-                        'PseudomonasAnnotation.tsv',
+    'annotation_file':  os.path.join(REPO_ROOT, 'data',
+                                     'PseudomonasAnnotation.tsv'),
 })
 
 # test server deployment credentials and configuration (maybe on AWS)
 TEST_CONFIG = {
     'data':          DATA_CONFIG,
     'host_conn':     AWS_CONFIG['host_conn'].copy(),
     'virt_env':      OS_CONFIG['home_dir'] + '.virtualenvs/adage',
-    'django_dir':    OS_CONFIG['home_dir'] + 'adage-server/adage',
+    'django_dir':    os.path.join(REPO_ROOT, 'adage'),
```

Comment:
Okay, these are the kinds of changes you wanted to test, right? I'll see if they break my development configuration.